aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/AddVersionCommand.java

```diff
@@ -150,7 +154,7 @@ public Integer call() {
         try {
             if (StringUtils.isBlank(applicationGuid)) {
                 log.info("Searching for application '{}' on AIP Console", applicationName);
-                applicationGuid = applicationService.getOrCreateApplicationFromName(applicationName, autoCreate, nodeName, domainName);
+                applicationGuid = applicationService.getOrCreateApplicationFromName(applicationName, autoCreate, nodeName, domainName, inPlaceMode);
```

Comment:
you need to check if the application is really in inplace mode, if the application is already created

Reply:
Check done earlier here
inplaceMode = app == null ? isInPlaceMode() : app.isInPlaceMode();